### PR TITLE
ECCW-548: Prevent strftime deprecation warnings on probe and upgrade to Drupal 9.5.9

### PR DESCRIPTION
**Pull request by MatthewWilkes**
The probe endpoint uses strftime, this applies a patch to switch to date to prevent it polluting logs.

This also upgrades to Drupal 9.5.9, with a security fix.

### CODE REVIEW

**Comment by MatthewWilkes** [comment]
@siliconmeadow I've just looked at composer lock, and it looks like it's pinned a dev version of Drupal core??

File: composer.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
         "drupal/config_split": "^2.0@RC",
         "drupal/core-composer-scaffold": "^9.1",
         "drupal/core-project-message": "^9.1",
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-recommended": "^9.1@stable",
```
Comment (siliconmeadow) [inline composer.json:47]: If you do a `ddev composer update "drupal/core-*" --with-all-dependencies` you'll get D9.5.9 and this line will remain the same. I don't quite know how different the `composer.lock` file will be when you do so. But the upgrade process for Drupal if using `drupal/core-recommended` is `composer update "drupal/core-*" --with-all-dependencies`. I always defer to this page: https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer

Comment (siliconmeadow) [reply composer.json:47]: I did one earlier! :-)

Comment (siliconmeadow) [reply composer.json]: In the end, the differences were trivial. Typical.
Cursor_and_git_diff_--cached

**Review by siliconmeadow — APPROVED** [review]
LGTM

**Pull request merged**